src/workerd/jsg/jsvalue.h

```diff
@@ -255,6 +255,7 @@ class JsString final: public JsBase<v8::String, JsString> {
 
   int hashCode() const;
 
+  bool isOneByte(Lock& js) const KJ_WARN_UNUSED_RESULT;
```

Comment:
Nit: `isOneByte` is super cheap. The `KJ_WARN_UNUSED_RESULT` is probably unnecessary.